README.md

```diff
@@ -263,6 +263,36 @@ The downloader supports going through a proxy when using the CloudCardPhotoServi
 - `TouchNetClient.originId`
     - provided by CloudCard
 
+#### Origo Storage Service Settings
+*Note: `downloader.storageService` must be set to `OrigoStorageService` for these to have any effect.*
+*Note: This integration does NOT YET support PKI authentication for Origo's production APIs.*
+*Visit Origo's Mobile Access documentation for more information https://doc.origo.hidglobal.com/api/*
+
+- `Origo.eventManagementApi`
+    - URI for accessing events in Mobile Access, such as user creation events.
+- `Origo.mobileIdentitiesApi`
+    - URI for user and photo related requests.
+- `Origo.certIdpApi`
+    - URI for Authentication requests.
+- `Origo.organizationId`
+    - Provided by origo. First half of clientId.
+- `Origo.accessToken`
+    - Temporary token obtained through authentication request. Only needs to be specified in certain development situations.
+- `Origo.clientSecret`
+    - System account password or PKI token. 
+    - Required for authentication.
+- `Origo.clientId`
+    - Provided by Origo.
+    - Available in Origo Management Portal.
+- `Origo.contentType`
+    - Used for request headers - see documentation.

```

Comment:
Is there a situation where this isn't always the same value?

Reply:
Not unless there was a change in API Version. Would you like these properties to stay filled?

Reply:
For the moment, I've included the properties that don't change in the existing application.properties file.

Reply:
Made a little mistake. While the content-type shouldn't change except for a couple requests (Already accounted for in OrigoClient methods), All of the API links change from test API to the Production API. I'll add the production API links to the Application.properties.